multi-int-store/src/main/java/com/connexta/multiintstore/config/ApplicationConfiguration.java

```diff
@@ -42,4 +43,16 @@ public EndpointUrlRetrieve endpointUrlRetrieve() {
   public RestTemplate restTemplate(RestTemplateBuilder builder) {
     return builder.errorHandler(new DefaultResponseErrorHandler()).build();
   }
+
+  @Bean
```

Comment:
❓ I didn't see any info about request bodies or request parameters either in the Actuator endpoint or form the app logs. Is this expected?

Reply:
It is expected to see bodies in the app logs. 
You will not see bodies Actuator.
I'll look into it when I get local dev working or when I back to the office to get access to the cloud env.

Reply:
I tried and tried to get the bodies logged. Never succeeded. The body is a stream, which you can only read once. There are caching classes to decorate the http filters that let you read the stream (up to limit) more than once. Couldn't get those to work. Could not even get the filter registered properly.